modules/gitops-release-notes-1-9-0.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-9-0_{context}"]
+= Release notes for {gitops-title} 1.9.0
+
+{gitops-title} 1.9.0 is now available on {product-title} 4.12 and 4.13.
+
+[id="new-features-1-9-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, you can use a custom `must-gather` tool to collect diagnostic information for project-level resources, cluster-level resources, and {gitops-title} components. This tool provides the debugging information about the cluster associated with {gitops-title}, which you can share with the Red Hat Support team for analysis. link:https://issues.redhat.com/browse/GITOPS-2797[GITOPS-2797]
++
+[IMPORTANT]
+====
+The custom `must-gather` tool is a Technology Preview feature.
+====
+
+* With this update, you can add support to progressive delivery using Argo CD Rollouts. Currently, the supported traffic manager is only {SMProductName}. link:https://issues.redhat.com/browse/GITOPS-959[GITOPS-959]
++
+[IMPORTANT]
+====
+Argo CD Rollouts is a Technology Preview feature.
+====
+
+[id="deprecated-features-1-9-0_{context}"]
+== Deprecated and removed features
+
+* In {gitops-title} 1.7.0,  the `.spec.resourceCustomizations` parameter was deprecated. The deprecated `.spec.resourceCustomizations` parameter is planned to be removed in the upcoming {gitops-title} GA v1.10.0 release. You can use the new formats `spec.ResourceHealthChecks`, `spec.ResourceIgnoreDifferences`, and `spec.ResourceActions` instead. link:https://issues.redhat.com/browse/GITOPS-2890[GITOPS-2890]
```

Comment:
minor nit: I'm wondering if there's a way to get rid of the double passive voice here: "parameter is planned to be removed". "removal is planned" maybe?

Reply:
@deerskindoll Thanks for the suggestion. But this is part of the template prescribed here that we follow: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts 

Search for and refer to the Deprecation notice heading. This is the way we write throughout all RNs in the OCP docs.

Reply:
@deerskindoll Thanks for the suggestion. But this is part of the template prescribed here that we follow: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts 

Search for and refer to the Deprecation notice heading. This is the way we write throughout all RNs in the OCP docs.